pkg/controller/argocd/rolebinding.go

```diff
@@ -61,7 +60,37 @@ func newRoleBindingWithname(name string, cr *argoprojv1a1.ArgoCD) *v1.RoleBindin
 	return roleBinding
 }
 
-func (r *ReconcileArgoCD) reconcileRoleBinding(name string, role *v1.Role, sa *corev1.ServiceAccount, cr *argoprojv1a1.ArgoCD) error {
+// reconcileRoleBindings will ensure that all ArgoCD RoleBindings are configured.
+func (r *ReconcileArgoCD) reconcileRoleBindings(cr *argoprojv1a1.ArgoCD) error {
+	if err := r.reconcileRoleBinding(applicationController, policyRuleForApplicationController(), cr); err != nil {
+		return err
+	}
+	if err := r.reconcileRoleBinding(dexServer, policyRuleForDexServer(), cr); err != nil {
+		return err
```

Comment:
Let's wrap these with messages, to make it easier to identify where the error occurred?

Reply:
I think we should wrap these with messages.